doc/feature/Zebra.md

```diff
@@ -0,0 +1,112 @@
+Restriping the output of a compressor for storage
+============
+
+This document specifies the transformation of the output of a data
+compressor into a smaller set of larger output chunks. The primary use
+case is as a backend of the memcached keyv::Map, which has a maximum
+value size of one megabyte.
+
+## Requirements
+
+* @subpage data
+* Input: data::CompressorInfo, uncompressed data, max output chunk size
+* Output: n output chunks of size <= max output chunk size
+* Output of uncompressed data if data is uncompressible with zero-copy
+  during compression and decompression
+
+and
+
+* Input: n output chunks
+* Output: uncompressed data
+
+## API
+
+    namespace pression
+    {
+    namespace data
+    {
+    class Zebra
+    {
+        struct Result { uint8_t* data; uint32_t size; };
+        typedef std::vector< Result > Results; //!< Set of result chunks
+        typedef std::vector< uint32_t > ResultSizes; //!< Remaining chunk sizes
+
+        Zebra( const CompressorInfo& compressor );
+
+        // returned pointers are valid until next compress(), delete of
+        // input data, or dtor of Zebra called
+        Results&& compress( const uint8_t* data, size_t size,
+                            uint32_t chunkSize );
+
+        // input: first chunk
+        ResultSizes&& getRemainingSizes( const uint8_t* data, uint32_t size );
+        size_t getDecompressedSize( const uint8_t* data, uint32_t size );
```

Comment:
Both functions will be documented in the .h during implementation. It's redundant here.